### ISSUE TITLE
Alt tag for images

### ISSUE THREAD

**Issue by hchouhan** [flowdee/amazon-product-in-a-post-plugin#22]
Currently, the alt tags for images is "Buy Now" and for a customer's website there is none http://whatdoipress.com/index.php/instant-pot-duo60-6-qt-7-in-1-multi-use-programmable-pressure-cooker/.

Is there any way to set a custom alt tag?

**Comment by hchouhan** [comment]
https://secure.helpscout.net/conversation/1325111217/30061?folderId=2352906

**Comment by webdj-agency** [comment]
@hchouhan  yes, there are 2 hooks for such cases. customer can add any of these lines inside of the active theme "functions.php" file:
```
add_filter( 'appip_alt_text_main_img', function() { return "CUSTOM_ALT_TEXT_FOR_IMAGES"; } );
add_filter( 'appip_amazon_button_alt_text', function() { return "CUSTOM_ALT_TEXT_FOR_BUTTONS"; } );
```

**Comment by hchouhan** [comment]
@webdj-agency once the filter is added, how exactly can a customer set custom alt tags within the shortcode?

**Comment by webdj-agency** [comment]
@hchouhan this filter is intended to set the simultaneous custom value for all the tags at once (in the functions.php file).
"CUSTOM_ALT_TEXT_FOR_IMAGES" and "CUSTOM_ALT_TEXT_FOR_BUTTONS" should be changed to the preferred value

**Comment by hchouhan** [comment]
@webdj-agency I don't have API keys so hard to test it in order to understand it.

Can you please tell me what happens to the code of images once these 2 filters are placed? Can you share some example code/screenshot of before and after placing this code.

**Comment by webdj-agency** [comment]
Sure! Here you go:
![filters](https://user-images.githubusercontent.com/64838430/100548693-d110c900-3276-11eb-8db3-e4a9b067bd74.png)
![before](https://user-images.githubusercontent.com/64838430/100548699-da019a80-3276-11eb-97ac-6c7e88e90655.png)
![after](https://user-images.githubusercontent.com/64838430/100548701-dcfc8b00-3276-11eb-8f1c-7280e4565710.png)

**Comment by hchouhan** [comment]
Hey @webdj-agency,

The client wants to either set custom alt tags or use the product title as the alt image. Is that possible?

**Comment by webdj-agency** [comment]
`use the product title as the alt image` is not implemented but it's possible to

**Comment by hchouhan** [comment]
@webdj-agency @flowdee can we work on this?

**Comment by flowdee** [comment]
I think, using the product title as default alt tag should be fine for now?

**Comment by webdj-agency** [comment]
Done. @flowdee  please check my pull request
